src/threads.py

```diff
@@ -6,9 +6,7 @@
 internal storages (like inventory and knownnodes) and do forced garbage
 collection, `singleWorker` for doing PoW, `sqlThread` for querying sqlite
 database.
-
```

Comment:
I'm not sure these changes are needed

Reply:
Okay. I also think this isn't needed. I Will amend the changes that I had done. Thank you